episodes/02-lessons.md

```diff
@@ -35,7 +40,7 @@ Make some notes, then share a single-sentence answer to each question with the o
 
 :::::::::::::::::::::::::::::::::::::::: keypoints
 
-- First key point. Brief Answer to questions. (FIXME)
+- There can be many reasons to create a new lesson. What is important is that the authors are excited about the project.
```

Comment:
I agree that whatever project we work on, if we are enthusiastic, we will work better.   But suppose we have to create the lesson because it's part of our job and it's on a topic we're not very excited about.  

This kind of training is very useful anyway, to do the best work possible and use the time efficiently (something like, since you have to do the work anyway, do it right.  If you use these tools you can have a better product in the same or less time).  

It seems worth mentioning or highlighting in some way, that these tools will allow you to create an effective lesson using your time efficiently.  You are going to generate a lesson that works because it is based on methods and tools that we know work because of the scientific evidence that supports these methods and tools.  

What do you think?

Reply:
These are good points, thank you @yabellini. I will rewrite this key point to try to capture them.

Reply:
I have tried to summarise better with f10a9b1. The point about the scientific evidence to back up the methods taught is better kept for the next episode, I think.